app/controllers/admin/notes_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Admin
+  class NotesController < ApplicationController
+    def create
+      @submission = Submission.find(note_params[:submission_id])
+      note = Note.new(gravity_user_id: @current_user, **note_params)
+
+      if note.save
+        redirect_to admin_submission_path(@submission),
+                    notice: 'Note has successfully been created.'
+      else
+        redirect_to admin_submission_path(@submission),
+                    alert:
+                      "Could not create note: #{
+                        note.errors.full_messages.join(', ')
+                      }"
+      end
```

Comment:
Some refactoring ideas for you:

```suggestion
      note_attrs = note_params.merge(gravity_user_id: current_user)
      note = Note.new(note_attrs)

       path = admin_submission_path(note.submission)

       if note.save
         redirect_to path, notice: 'Note has successfully been created.'
      else
        error_messages = "Could not create note: #{note.errors.full_messages.join(', ')}"
        redirect_to path, alert: error_messages
      end
```

Take or leave any/all of these ideas! 🤗